Sources/TensorFlow/Layers/Core.swift

```diff
@@ -223,3 +223,21 @@ public extension Dense {
                   activation: activation)
     }
 }
+
+/// A `Function` layer arbitrarily stores any differentiable function and returns its
```

Comment:
This comment can be improved for accuracy. The function layer does _not_ "return its `callAsFunction(_:)` method".

I would suggest:
```
/// A layer that encloses a custom differentiable function.
```

Reply:
Oh that sounds very neat, I shall update it.